Source/EasyNetQ.Management.Client.ApprovalTests/EasyNetQ.Management.Client.approved.txt

```diff
@@ -1,1082 +1,1093 @@
-        public string Exchange { get; set; }
-        public string Read { get; set; }
-        public string User { get; set; }
-        public string Vhost { get; set; }
-        public string Write { get; set; }
-    }
-    public class TopicPermissionInfo : System.IEquatable<EasyNetQ.Management.Client.Model.TopicPermissionInfo>
-    {
-        public TopicPermissionInfo(string UserName, string? Exchange = null, string Write = ".*", string Read = ".*") { }
-        public string? Exchange { get; set; }
-        public string Read { get; set; }
-        public string UserName { get; set; }
-        public string Write { get; set; }
-        public EasyNetQ.Management.Client.Model.TopicPermissionInfo DenyAllRead() { }
-        public EasyNetQ.Management.Client.Model.TopicPermissionInfo DenyAllWrite() { }
-        public EasyNetQ.Management.Client.Model.TopicPermissionInfo SetExchange(string? exchange) { }
-        public EasyNetQ.Management.Client.Model.TopicPermissionInfo SetRead(string resourcesToAllow) { }
-        public EasyNetQ.Management.Client.Model.TopicPermissionInfo SetWrite(string resourcesToAllow) { }
-    }
-    public class User : System.IEquatable<EasyNetQ.Management.Client.Model.User>
-    {
-        public User(string Name, string PasswordHash, System.Collections.Generic.IReadOnlyList<string> Tags) { }
-        public string Name { get; set; }
-        public string PasswordHash { get; set; }
-        public System.Collections.Generic.IReadOnlyList<string> Tags { get; set; }
-    }
-    public class UserInfo : System.IEquatable<EasyNetQ.Management.Client.Model.UserInfo>
-    {
-        public UserInfo(string Name, string? Password, string? PasswordHash, System.Collections.Generic.IReadOnlyList<string> Tags) { }
-        public string Name { get; set; }
-        public string? Password { get; set; }
-        public string? PasswordHash { get; set; }
-        public System.Collections.Generic.IReadOnlyList<string> Tags { get; set; }
-        public EasyNetQ.Management.Client.Model.UserInfo AddTag(string tag) { }
-        public static EasyNetQ.Management.Client.Model.UserInfo ByPassword(string name, string password) { }
-        public static EasyNetQ.Management.Client.Model.UserInfo ByPasswordHash(string name, string passwordHash) { }
-    }
-    public static class UserTags
-    {
-        public const string Administrator = "administrator";
-        public const string Impersonator = "impersonator";
-        public const string Management = "management";
-        public const string Monitoring = "monitoring";
-        public const string Policymaker = "policymaker";
-    }
-    public class Vhost : System.IEquatable<EasyNetQ.Management.Client.Model.Vhost>
-    {
-        public Vhost(string Name, bool Tracing = false) { }
-        public string Name { get; set; }
-        public bool Tracing { get; set; }
-    }
+namespace EasyNetQ.Management.Client

```

Comment:
I don't know why, but this file was completely replaced: git detects no changes, but shows all lines of the file as new.

Reply:
I've checked file encoding and special characters, it seems to me there are no significant modifications

Reply:
Ok, no worries then :)